features/gezag-minderjarige/1.1-ingezetene-BRP.feature

```diff
@@ -0,0 +1,122 @@
+
+    Scenario: minderjarige staat ingeschreven in RNI (route 1)
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * is minderjarig
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de RNI
+      | gemeente van inschrijving (09.10) |
+      | 1999                              |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                               |
+      | type        | GezagNietTeBepalen                                                   |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet in Nederland woont. |
+
+    Scenario: minderjarige staat ingeschreven in Nederlandse gemeente
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * is minderjarig
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      | gemeente van inschrijving (09.10) |
+      | 0518                              |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+
+  Regel: onderzoek op gemeente van inschrijving leidt niet tot uitval
+
+    Abstract Scenario: minderjarige staat ingeschreven in Nederlandse gemeente en <omschrijving> staat in onderzoek
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * is minderjarig
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
```

Comment:
Ook dit zou ik omkeren. Dit is de standaard situatie en als er geen verblijfplaats is vastgelegd (scenario op regel 107) zou ik dat expliciet benoemen

Reply:
Bij andere features zou ik het daarmee misschien eens zijn. Deze feature gaat specifiek over de inschrijving in BRP dan wel RNI. Dan moet je m.i. die inschrijving in BRP wel expliciet maken en daar zelfs bij opnemen dat dit betekent dat gemeente van inschrijving een waarde ongelijk aan 1999 heeft, zoals ik gedaan heb in scenario "minderjarige staat ingeschreven in Nederlandse gemeente".

@CathyDingemanse in hoeverre wil je dat iemand die het stroomschema kent door de stappen/vragen van het stroomschema moet kunnen lezen in de/alle scenario's, zodat die het verwachtte antwoord begrijpt? Bijvoorbeeld:
* is ingeschreven in de BRP
* is minderjarig
* is niet geëmigreerd geweest
* is in Nederland geboren
* heeft geen uitspraak gezag
* heeft 2 juridische ouders
* beide ouders zijn nu met elkaar gehuwd
* beide ouders zijn bevoegd tot gezag